Source/Scene/Primitive.js

```diff
@@ -1077,31 +1103,32 @@ define([
     function createShaderProgram(primitive, frameState, appearance) {
         var context = frameState.context;
```

Comment:
Yes, it's only `GroundPrimitive`. Since the polygons are draped on terrain, they don't need to cast shadows. If they are translucent, they probably don't need to receive shadows, but I would test to see how it looks when you have terrain receive shadows.